test/jdk/com/sun/crypto/provider/Cipher/AEAD/GCMBufferTest.java

```diff
@@ -128,7 +134,9 @@
     GCMBufferTest(String algo, List<dtype> ops) {
         this.algo = algo;
         this.ops = ops;
-        theoreticalCheck = true;
+        theoreticalCheckUpdate = false; // Disabling length verification after cipher.update for 
+                                        // GCM Buffer Tests, the variable is never set to true
```

Comment:
I think a better place for the comments is with the field declarations.

Reply:
The comment text contains what I wanted to see (the variable is never set to true). I'm fine with having it on the declaration.

Reply:
Moved comment to the declaration.